typings/modifiers.d.ts

```diff
@@ -0,0 +1,34 @@
+import {colorsPalette} from '../src/style/colorsPalette';
```

Comment:
actually you are right. initially, this file contain lots of manual types with all the combination of the dynamic usage, for example "paddingT-s3". as you can see, the problem is with the s3, we can't automatically generate typings for the dynamic value, so the former guy that create the manual types, added predefined values for many of the dynamic ones and I wanted to use them. but then I found out that typescript can't read the '-' symbol, so it didn't help us and I removed it. 
Now I can put everything inside the modifier.ts

Reply:
Since we will never support dynamic modifiers values (for instance if someone load his own colors or spacings values) 
So maybe it's ok not to have typings for that.. I don't know. 
the question is, if user will use the component with a modifier that has no typing 
`<View bg-customColor .../>`
will they get a tslint error?

Reply:
when I say dynamic, I mean the value. when you type paddingT-785, the value is dynamic and typescript currently don't have support for patterns, so we can't verify that the value is good. as a workaround, the dude from Kiev added some typings for common values, like `padding-5 | padding-6 | padding-7| etc...` but it doesn't work, because as I said, it looks like typescript can't handle the `-`.